templates/certificate_autorenewal_configuration.yaml.erb

```diff
@@ -0,0 +1,23 @@
+# Managed by Puppet
+# puppet-sslcert
+---
+auto_cert_renewal_certificates:
+        <%= @name %>:
```

Comment:
Veel indent hier? is 2 spaties niet genoeg?

Reply:
ik zal het aanpassen